### PR TITLE
Serve front

### PR DESCRIPTION
**Pull request by BourgoisMickael**
Moved api at root and front files in a front folder.

API serves all routes in front folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,86 @@
-Website for Raven-OS
+# Website's API
```
Comment (Arignir) [inline README.md]: Title and first line should be updated to talk about the whole website, and not only it's backend.

File: front/404.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      Theme URL: https://bootstrapmade.com/regna-bootstrap-onepage-template/
+      Author: BootstrapMade.com
+      License: https://bootstrapmade.com/license/
+    ======================================================= -->
+</head>
+<body>
+  <section id="hero">
+      <div class="hero-container">
+          <h1>404</h1>
+          <h2>NOT FOUND LOL</h2>
```
Comment (Arignir) [inline front/404.html]: Could we be a bit more serious ? x)

(Title too)

File: front/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Website for Raven-OS
```
Comment (Arignir) [inline front/README.md]: Looks unnecessary for so little informations

**Comment by Arignir** [comment]
Only a few changes are required, this shouldn't take long.

**Comment by Arignir** [comment]
I'd like to test this before merging.

I'll try it tonight.

File: front/footer.html.hbs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        <div class="copyright">
+            &copy; Copyright <strong>Raven</strong>. All Rights Reserved
+        </div>
+        <div class="credits">
+            <!--
+              All the links in the footer should remain intact.
+              You can delete the links only if you purchased the pro version.
+              Licensing information: https://bootstrapmade.com/license/
+              Purchase the pro version with working PHP/AJAX contact form: https://bootstrapmade.com/buy/?theme=Regna
+            -->
```
Comment (person-m) [inline front/footer.html.hbs]: Do we really need this comment ?

File: src/catchers.rs [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+use std::collections::HashMap;
+
+/// Returns all error catchers
+pub fn get_catchers() -> Vec<Catcher> {
+    errors![not_found]
+}
+
+#[error(404)]
+fn not_found(_req: &Request) -> Template {
+    //println!("{:?}", _req);
```
[the remark this answers is not in the document]
Comment (BourgoisMickael) [reply src/catchers.rs]: done

File: front/footer.html.hbs (hunk 4 above)
Comment (BourgoisMickael) [reply front/footer.html.hbs]: removed

File: front/.gitignore [hunk 6]
```diff
@@ -0,0 +1,64 @@
+# Windows image file caches
```
[the remark this answers is not in the document]
Comment (BourgoisMickael) [reply front/.gitignore]: removed

**Pull request merged**